demo-project/src/demo_project/pipelines/data_ingestion/nodes.py

```diff
@@ -96,7 +96,7 @@ def aggregate_company_data(typed_companies: pd.DataFrame) -> pd.DataFrame:
 
 
 def combine_shuttle_level_information(
-    shuttles: pd.DataFrame, companies: pd.DataFrame, reviews: pd.DataFrame
+    shuttles: pd.DataFrame, shuttles2:pd.DataFrame, companies: pd.DataFrame, reviews: pd.DataFrame
```

Comment:
Is there a more descriptive name we can use than `shuttles2`? Having `shuttles` and `shuttles2` is inconsistent and doesn't help me distinguish why there are two.

Reply:
hey Jannic, i mentioned in my QA notes, this is just for front-end test purpose. and this wont be merged. Infact i will remove it now since Tynan approved the fe bit